src/Mint.js

```diff
@@ -0,0 +1,354 @@
+				setErrorMessage('It looks like there was a connection issue. Please refresh the page and try again. If this issue persists please reach out to the team on Discord and we will get back to you ASAP!');
+				setShowNote(true);
+				console.log("Wallet not connected");
+			}
+		}
+		catch (e) {
+			setErrorMessage('It looks like there was an error minting! Please make sure you have sufficient funds in your wallet. If this issue persists please reach out to the team on Discord and we will get back to you ASAP!');
+			setShowNote(true);
+			console.error(e);
+		}
+	}
+
+	const mintButton = saleStarted ? (
+		<>
+			<button
+				className="block w-full py-3 px-4 rounded-md shadow bg-indigo-500 text-white font-medium hover:bg-indigo-600 focus:outline-none focus:ring-2 focus:ring-offset-2 focus:ring-indigo-300 focus:ring-offset-gray-900"
+				onClick={() => mint(numTokensToPurchase)}
+			>
+				Mint
+			</button>
+			<p>
+				Mint {numTokensToPurchase} Faces for {(tokenPrice * numTokensToPurchase) / (10 ** 18)} ETH + GAS
+			</p>
+		</>
+	) : (
+		<button
+			className="opacity-50 block w-full py-3 px-4 rounded-md shadow bg-indigo-500 text-white font-medium"
+		>
+			Sale has not started	
+		</button>
+	)
+
+	return (
+		<div className="min-h-screen">
+			<div className="relative overflow-hidden">
+			<main>
+				<div className="pt-10 sm:pt-16 lg:pt-8 lg:pb-14 lg:overflow-hidden">
+					<div className="mx-auto max-w-7xl lg:px-8">
+						<div className="lg:grid lg:grid-cols-2 lg:gap-8">
+							<div className="mx-auto max-w-md px-4 sm:max-w-2xl sm:px-6 sm:text-center lg:px-0 lg:text-left lg:flex lg:items-center">
+								<div className="lg:py-24">
+									<h1 className="mt-4 text-4xl tracking-tight font-extrabold sm:mt-5 sm:text-6xl lg:mt-6 xl:text-6xl">
+										<span className="block">Mint a Face Token</span>
+									</h1>
+									<div className="mt-10 sm:mt-12 h-36">
+										{walletAddress ? (
+										<button
+											className="block w-full py-3 px-4 rounded-md shadow bg-green-500 text-white font-medium focus:outline-none focus:ring-2 focus:ring-offset-2 focus:ring-indigo-300 focus:ring-offset-gray-900 mb-3"
+										>
+											Connected
+										</button>
+										) : (
+										<button
+											className="block w-full py-3 px-4 rounded-md shadow bg-indigo-500 text-white font-medium hover:bg-indigo-600 focus:outline-none focus:ring-2 focus:ring-offset-2 focus:ring-indigo-300 focus:ring-offset-gray-900 mb-3"
+											onClick={signIn}
+										>
+											Connect Wallet
+										</button>
+										)}
+										<div className="sm:flex">
+											{saleStarted && <div className="min-w-0 flex-1">
+												<label htmlFor="text" className="sr-only">
+													Number of tokens to mint
+												</label>
+												<div className="relative rounded-md">
+													<input
+														id="num-tokens"
+														type="text"
```

Comment:
should this be type number?

Reply:
Good catch! Setting this type to `number` adds extra validation to only allow number